client/src/components/index.js

```diff
@@ -16,3 +16,4 @@ export { default as Spotlight } from './Spotlight';
 export { default as Teachers } from './Teachers';
 export { default as AboutUs } from './AboutUs';
 export { default as Footer } from './Footer';
+export { default as LandingPageNews } from './LandingPageNews';
```

Comment:
Any reason why you're not also adding NewsTile as well?

Reply:
I expect that we'll only be using the news tile for the landing page news component. I don't think we'll need to use it elsewhere. I can add it though if you think it's a good idea!

Reply:
np, was just curious!